tensorflow/contrib/distributions/python/kernel_tests/beta_test.py

```diff
@@ -293,6 +293,36 @@ def testBetaWithSoftplusAB(self):
       self.assertAllClose(tf.nn.softplus(a).eval(), dist.a.eval())
       self.assertAllClose(tf.nn.softplus(b).eval(), dist.b.eval())
 
+  def testBetaBetaKL(self):
```

Comment:
can you run these tests with bazel --runs_per_test=500 and ensure they never fail?